reference/catalog.v3.yml

```diff
@@ -3816,7 +3816,7 @@ paths:
         **Read-Only Fields**
         * id
 
-        **Note:** The max number of metafields allowed on each product, category, variant, or brand is 250 per client ID. For more information, see [Platform Limits](https://support.bigcommerce.com/s/article/Platform-Limits) in the Help Center.
+        **Note:** The maxiumum number of metafields allowed on each order, product, category, variant, or brand is 250 per client ID. For more information, see [Platform Limits](https://support.bigcommerce.com/s/article/Platform-Limits) in the Help Center.
```

Comment:
@traci707 @bookernath so a product, etc, updated by multiple API keys could have (250 * x) metafields, where x is the number of API keys in the mix? Seems weird.

Reply:
Correct. The limit is per-entity-per-client-ID. This is an area where I think it's best that clients don't have to worry about the activity of other clients messing up their ability to use the system - it would be pretty frustrating if I have an app that relies on metafields, and I get some error related to "you can't add metafields to this product because other clients already added too many".

Metafields are pretty simple (key/value storage/retrieval) and the endpoints are paginated, so we're not too worried about it so far.